packages/web/src/dev/ValidatorsList.tsx

```diff
@@ -0,0 +1,623 @@
+import { BigNumber } from 'bignumber.js'
+import * as React from 'react'
+import { StyleSheet, Text as RNText, View } from 'react-native'
+import CopyToClipboard from 'src/dev/CopyToClipboard'
+import ProgressCutBar from 'src/dev/ProgressCutBar'
+import { H1 } from 'src/fonts/Fonts'
+import { I18nProps, withNamespaces } from 'src/i18n'
+import Chevron, { Direction } from 'src/icons/chevron'
+import { HEADER_HEIGHT } from 'src/shared/Styles'
+import { colors, standardStyles, textStyles, typeFaces } from 'src/styles'
+import { cutAddress, formatNumber, weiToDecimal } from 'src/utils/utils'
+
+class Text extends RNText {
+  render() {
+    return <RNText style={[styles.defaultText, this.props.style]}>{this.props.children}</RNText>
+  }
+}
+
+interface HeaderCellProps {
+  style: any[]
+  name: string
+  order: boolean | null
+  onClick: () => void
+}
+
+const HeaderCell = React.memo(function HeaderCellFn({
+  style,
+  name,
+  order,
+  onClick,
+}: HeaderCellProps) {
+  return (
+    <View onClick={onClick} style={[styles.tableHeaderCell, ...((style || []) as any)]}>
+      <Text>{name}</Text>
+      <Text
+        style={[
+          styles.tableHeaderCellArrow,
+          ...(order !== null ? [styles.tableHeaderCellArrowVisible] : []),
+        ]}
+      >
+        <Chevron direction={order ? Direction.up : Direction.down} color={colors.white} size={10} />
+      </Text>
+    </View>
+  )
+})
+
+interface ValidatorsListProps {
+  data: any
+  isLoading: boolean
+}
+
+export interface State {
+  expanded: number | undefined
+  orderBy: string
```

Comment:
if there are only a limited set of strings that it can be ordered by its nice to list them like 

`orderedBy: 'name' | 'votes'`